test/runner.js

```diff
@@ -1812,6 +1812,78 @@ describe('Runner', () => {
         expect(notebook.failures).to.equal(1);
     });
 
+    it('passes shallow clones of context to tests', async () => {
+
+        const script = Lab.script({ schedule: false });
+
+        const testContext = {
+            testData: { hello: 'there' }
+        };
+
+        const innerContext = {
+            testData: { goodbye: 'you' },
+            additionalData: { another: 'object' }
+        };
+
+        script.experiment('test', () => {
+
+            script.before(({ context }) => {
+
+                context.testData = testContext.testData;
+            });
+
+            script.test('has test context', ({ context }) => {
+
+                expect(context,'has proper context').to.equal(testContext);
+                expect(context,'is a shallow clone').to.not.shallow.equal(testContext);
```

Comment:
I think to really test this you would have to use a separate variable to assign the `context` passed into before, then compare the test's `context` against that.  In short: even without shallow cloning this would pass.

Reply:
that assertion is only there to prove it's not a direct reference to the original testContext, i agree it's a bit weird but i wanted to make sure that was captured. if it's deep equal but not shallow equal, then it must be a shallow clone right?

Reply:
ahhhh yes, ok i see what you're saying. i'll tweak things to compare the context object in each location directly and remove the nonsense equality assertions that are here now